gpii/node_modules/gpii-iniFile/README.md

```diff
@@ -0,0 +1,165 @@
+[GPII/ini](https://github.com/GPII/ini), where the required changes where too significant for its original charm to
+remain. So, while looking roughly the same from the outside, this is a re-write rather than a fork.
+
+## INI file format
+
+INI file can look something like this:
+
+```ini
+# comments
+; other comments
+
+# = or : delimiters.
+key=value1
+key2:value2
+key3 = value3
+
+# Sections (and sub-sections)
+[section]
+key="section-value"
+
+[[subsection]]
+key=subsection value
+
+[section2]
+# Multiple lines
+multi-line="""line1
+line2
+line3"""
+multi-line2=line1
+    line2
+    line3
+```
+
+Which will parse to the following:
+
+```javascript
+{
+    key: "value1",
+    key2: "value2",
+    key3: "value3",
+    section: {
+        key: "section-value",
+        subsection: {key: "subsection value"}
+    },
+    section2: {
+        "multi-line": "line1\nline2\nline3",
+        "multi-line2": "line1\nline2\nline3"
+    }
+}
+```
+
+See [test/read-test.ini](test/read-test.ini) for an extreme example.
+
+When writing to an existing file, the content is modified rather than re-writing. Effort is made to respect the
+unchanged values and current format of the file. An updated version of the above example will look like this:
+
+```ini
+# comments
+; other comments
+
+# = or : delimiters.
+key=new value
+key2:another new value
+key3 = and again
+
+# Sections
+[section]
+key="quotes respected"
+# sub-sections
+[[subsection]]
+key=subsection value
+
+new1=new value
+[section2]
+# Multiple lines
+multi-line="""modified line1
+modified line2
+modified line3"""
+multi-line2=modified line1
+    modified line2
+    modified line3
+
+[newSection]
+newValue=value
+```
+
+Compare [test/write-test.ini](test/write-test.ini) with [test/write-test.expect.ini](test/write-test.expect.ini) for an
+example.
+
+## Usage
+
+### Reading
+
+```javascript
+/**
+ * Reads INI file content, returning the parsed data as an object.
+ *
+ * @param {String} content The ini file content.
+ * @param {Object} options Options:
+ * @param {boolean} options.strings true to always return strings, otherwise try to return numbers and booleans for
+ *  unquoted values, where appropriate.
+ * @param {boolean} options.firstDuplicate true to use the first occurrence of a value with a duplicate name, otherwise
+ * the last is used. [default: false]
+ *
+ * @return {Object} The parsed ini file data.
+ */
+gpii.iniFile.read = function (content, options)
+```
+
+
+### Writing
+
+```javascript
+/**
+ * Writes an object to existing INI file content.
+ *
+ * @param {String} input The ini file content to update (can be empty).
+ * @param {Object} data The new settings data.
+ * @param {Object} options Options, mostly for writing new values.
+ * @param {String} options.keyValueDelimiter - Text which separates keys and values, for new values. [default: "="].
+ * @param {boolean} options.keepUndefined - true to keep items and sections that are not in data. Use to define values
```

Comment:
Adjust this to agree with the actual comment

Reply:
fixed